src/components/measurements/measurementsD3.js

```diff
@@ -194,19 +194,21 @@ export const drawMeasurementsSVG = (ref, xAxisRef, svgData) => {
   // x-axis is in a different SVG element to allow sticky positioning
   drawStickyXAxis(xAxisRef, containerHeight, svgHeight, xScale, x_axis_label);
 
-  // Add threshold if provided
-  if (threshold !== null) {
-    const thresholdXValue = xScale(threshold);
-    svg.append("line")
-      .attr("class", classes.threshold)
-      .attr("x1", thresholdXValue)
-      .attr("x2", thresholdXValue)
-      .attr("y1", layout.topPadding)
-      .attr("y2", svgHeight)
-      .attr("stroke-width", layout.thresholdStrokeWidth)
-      .attr("stroke", layout.thresholdStroke)
-      // Hide threshold by default since another function will toggle display
-      .attr("display", "none");
+  // Add threshold(s) if provided
+  if (thresholds !== null) {
+    thresholds.forEach((threshold, index) => {
```

Comment:
_non-blocking_

FWIW, you can do

```suggestion
    for (let threshold of thresholds) {
```

and not have to create a closure.

Reply:
Ah, thanks for pointing this out! It also made me realize that I no longer need the `index` value here.

Reply:
Updated in force-push.